rtos/TARGET_CORTEX/rtx5/mbed_rtx_conf.h

```diff
@@ -24,6 +24,9 @@
 
 #include "mbed_rtx.h"
 
+/** Any access to RTX5 specific data structures used in common code should be wrapped in ifdef MBED_OS_BACKEND_RTX5 */
```

Comment:
I have some doubts about that, I do get it why it's here, but we are pretending it will work in both cases, but in reality it won't have the same behavior.